src/ui/server/helpers/required-fields/eligibility/index.ts

```diff
@@ -33,7 +33,6 @@ export const irrelevantFields = [
  * Therefore, we do not want these fields in the list of required eligibility fields.
  * 1) ACCOUNT_TO_APPLY_ONLINE field - has no data/value.
  * 2) BUYER_COUNTRY_ISO_CODE field - data is saved as BUYER_COUNTRY relationship object.
- * 3) WANT_COVER_OVER_MAX_AMOUNT field - data is saved as TOTAL_CONTRACT_VALUE relationship object.
```

Comment:
Can replace this with a comment for `TOTAL_CONTRACT_VALUE` (saved as `TOTAL_CONTRACT_VALUE_ID`)?